package.json

```diff
@@ -7,6 +7,7 @@
     "bundle-data": "node scripts/bundle-data.js data/ docs/",
     "bundle:android": "react-native bundle --entry-file index.android.js --dev true --platform android --bundle-output ./android/app/src/main/assets/index.android.bundle --assets-dest ./android/app/src/main/res/",
     "bundle:ios": "react-native bundle --entry-file index.ios.js --dev true --platform ios --bundle-output ./ios/AllAboutOlaf/main.jsbundle --assets-dest ./ios",
+    "check:": "npm run prettier && npm run flow && npm run lint && npm run test",
```

Comment:
We don't need to run both "npm run prettier" and "npm run lint" – eslint is run as part of the prettier command.